doc/conf_ref.rst

```diff
@@ -222,6 +222,19 @@ or :ref:`mirrorlist <mirrorlist-label>` option definition.
     RPM debug scriptlet output level. One of: ``critical``, ``emergency``,
     ``error``, ``warn``, ``info`` or ``debug``. Default is ``info``.
 
+.. _search_enabled_metadata-label:
+
+``search_enabled_metadata``
```

Comment:
@jsilhan PackageKit already does have this option. It uses it to get the metadata so it shows up in search and whatnot without actually having the repo enabled.